### ISSUE TITLE
Use releases instead of commits to push updates to Home Assistant

### ISSUE THREAD

**Issue by eveiscoull** [TegridyTate/Hassarr#5]
Hey, fantastic integration! Such a handy thing to have.

However people will be bombarded with updates without release notes or a version number because you're not using the Releases feature of GitHub.

For example, the latest commit (labelled 156a5bb) is the name of the update in Home Assistant, and doesn't show the release note:

![Screenshot_20241226-195739.png](https://github.com/user-attachments/assets/77331020-5767-41cc-9444-e61b46faeb37)

Here's an example of what they typically look like (note that this is it downgrading a different integration from an alpha version to a production one):

![Screenshot_20241226-200339.png](https://github.com/user-attachments/assets/b05bf50b-71e4-4ea7-9544-76a0d4077ed4)

Now, I'm not a developer by any means and am unsure how to get that working, but this is the HACS documentation discussing it in minor detail: https://www.hacs.xyz/docs/publish/integration/#github-releases-optional

**Comment by TegridyTate** [comment]
Yep, I'm aware. Here's a comment about it: https://github.com/TegridyTate/Hassarr/issues/3#issuecomment-2562938635

I'm planning to work on it a bit on Sunday
